vmm/src/device_manager/legacy.rs

```diff
@@ -9,7 +9,7 @@ use std::io::{self, stdout};
 use std::sync::{Arc, Mutex};
 
 use devices;
-use sys_util::{self, EventFd, Terminal};
+use sys_util::{EventFd, Terminal};
 
 /// Errors corresponding to the `LegacyDeviceManager`.
 #[derive(Debug)]
```

Comment:
Ideally we will have a custom implementation for Debug & Display for these errors. That was the initial reason for deciding to use std::io::Error instead of errno.rs.